spec/controllers/idv/hybrid_handoff_controller_spec.rb

```diff
@@ -96,21 +96,46 @@
 
   describe '#update' do
     let(:analytics_name) { 'IdV: doc auth upload submitted' }
-    let(:analytics_args) do
-      { success: true,
-        errors: {},
-        destination: :link_sent,
-        flow_path: 'hybrid',
-        step: 'upload',
-        analytics_id: 'Doc Auth',
-        irs_reproofing: false,
-        skip_upload_step: false }
+
+    context 'hybrid flow' do
+      let(:analytics_args) do
+        { success: true,
+          errors: { message: nil },
+          analytics_id: 'Doc Auth',
+          destination: :link_sent,
+          flow_path: 'hybrid',
+          irs_reproofing: false,
+          step: 'upload',
+          telephony_response: { errors: {},
+                                message_id: 'fake-message-id',
+                                request_id: 'fake-message-request-id',
+                                success: true } }
+      end
+
+      it 'sends analytics_submitted event for hybrid' do
+        put :update, params: { doc_auth: { phone: '202-555-5555' } }
+
+        expect(@analytics).to have_logged_event(analytics_name, analytics_args)
+      end
     end
 
-    it 'sends analytics_submitted event' do
-      put :update, params: { doc_auth: { phone: '202-555-5555' } }
+    context 'desktop flow' do
+      let(:analytics_args) do
+        { success: true,
+          errors: {},
+          destination: :document_capture,
+          flow_path: 'standard',
+          step: 'upload',
+          analytics_id: 'Doc Auth',
+          irs_reproofing: false,
+          skip_upload_step: false }
+      end
```

Comment:
nitpicks:
1. preserve the key order in `analytics_args` assignment for easier comparison across `context`s.
2. standardize the indentation:
```
{
  success: true,
  errors: { message: nil },
  analytics_id: 'Doc Auth',
  destination: :link_sent,
  flow_path: 'hybrid',
  irs_reproofing: false,
  step: 'upload',
  telephony_response: {
    errors: {},
    message_id: 'fake-message-id',
    request_id: 'fake-message-request-id',
    success: true
  }
}
```

Reply:
Good point. Addressed in bf0985d5bf08a3599345172a7b58c617c343bd5c